package.json

```diff
@@ -32,8 +32,8 @@
   "devDependencies": {
     "angular-mocks": "1.4.7",
     "grunt": "0.4.5",
-    "grunt-angular-templates": "0.5.7",
     "grunt-contrib-cssmin": "0.12.2",
+    "grunt-angular-templates": "^1.0.3",
```

Comment:
sorry to be picky, but could you fix the version instead of having the latest one? Thanks